lang/java/reef-io/src/main/java/org/apache/reef/io/data/loading/impl/EvaluatorRequestSerializer.java

```diff
@@ -27,7 +27,9 @@
 /**
  * Serialize and deserialize EvaluatorRequest objects.
  * Supports number, memory, cores, nodeNames and rackNames serialization
+ * @deprecated in 0.12. Use AvroEvaluatorRequestSerializer instead
  */
+@Deprecated
 public final class EvaluatorRequestSerializer {
```

Comment:
I believe it is safe to delete this class. It was only ever used internally to the data loading service.